frameworks/agno/from-agno/agents/FinanceAgent/agent.py

```
@@ -0,0 +1,133 @@
+        self.add_datetime_to_instructions = True
+        self.show_tool_calls = True
+        self.markdown = True
+
+    async def run(self, request: AgentRequest, response: AgentResponse, context: AgentContext):
+        try:
+            user_input = (await request.data.text()).strip()
+            context.logger.info(f"Received input: {user_input}")
+
+            parsed = await parse_user_query(user_input)
+            tickers = parsed.get("tickers", ["AAPL"])
+            intent = parsed.get("intent", "market outlook")
```

Ruff (0.11.9)

Local variable `intent` is assigned to but never used

Remove assignment to unused variable `intent`

(F841)